string-manipulation-1/src/capitalize.js

```diff
@@ -0,0 +1,19 @@
+/* exported capitalize */
+
+// Check if first word is upper case
+// if lowercase make upper case
+// Check if there are more than one letter
+// if more letters, make all lower case
+// if
+
+function capitalize(word) {
+  var str = '';
+  for (var i = 0; i < word.length; i++) {
+    if (i === 0) {
+      str += word[i].toUpperCase();
+    } else {
+      str += word[i].toLowerCase();
+    }
```

Comment:
Instead of having a conditional in your loop that needs to run every iteration, you should just add the Uppercase letter before the loop begins, then start your loop at the `1` index

Reply:
Thanks for the feedback, fixed!